src/ngAria/aria.js

```diff
@@ -377,6 +377,25 @@ ngAriaModule.directive('ngShow', ['$aria', function($aria) {
             });
           }
         }
+
+        if (elem[0].nodeName === 'A') {
+          if ($aria.config('bindRoleForClick') && !attr.href && !attr.xlinkHref && !attr.role) {
+              elem.attr('role', 'link');
+          }
+          if ($aria.config('bindKeypress')) {
+            elem.on('keypress', function(event) {
+              var keyCode = event.which || event.keyCode;
+              var hasHref = (attr.href || attr.xlinkHref);
+              if ((keyCode === 32 || (keyCode === 13 && !hasHref)) && !attr.ngKeypress) {
+                elem[0].click();
```

Comment:
@LeeAdcock, what did you have in mind about this change ? In what way is this preferrable to just calling `fn` ?
BTW, if this ends up sticking around, then the callback below should be removed (as mentioned by @campersau).